packages/datagateway-search/src/searchPageContainer.component.tsx

```diff
@@ -444,11 +548,15 @@ const SearchPageContainer: React.FC<SearchPageContainerCombinedProps> = (
                       <SearchPageCardView
                         containerHeight={containerHeight}
                         hierarchy={match.params.hierarchy}
+                        onCurrentTab={pushCurrentTab}
```

Comment:
`onCurrentTab` is a little confusing a name for me - could this be `onCurrentTabChange` or `onTabChange` to indicate that this prop is the function that should be called when tabs change?